tests/unit/amo/components/TestPaginate.js

```diff
@@ -186,6 +186,32 @@ describe(__filename, () => {
     );
   });
 
+  describe('when receivedPageCount is passed', () => {
```

Comment:
This test looks good, but it doesn't really need to be wrapped in its own `describe`. You could just rename the test to `it('uses receivedPageCount for paginator links when passed', () => {`

Reply:
Done, thanks for the feedback!